kubectl-volsync/cmd/migration_create.go

```diff
@@ -36,18 +48,29 @@ var migrationCreateCmd = &cobra.Command{
 	and it sets up an associated ReplicationDestination that will be configured
 	to accept incoming transfers via rsync over ssh.
 	`)),
-	RunE: doMigrationCreate,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		r := &migrationCreate{
+			Command: *cmd,
+		}
+		return r.Run()
+	},
 	Args: validateMigrationCreate,
 }
 
 func init() {
 	migrationCmd.AddCommand(migrationCreateCmd)
 
-	migrationCreateCmd.Flags().String("accessmodes", "", "AccessModes of the PVC to create")
-	migrationCreateCmd.Flags().String("capacity", "", "capacity of the PVC to create")
+	migrationCreateCmd.Flags().String("copymethod", "",
+		"Copy method used to copy the data from source to destination PVC")
```

Comment:
In `copymethod = Direct`, we just copy data to destination PVC and don't create a snapshot of the PVC?

Reply:
Allowing the user to specify that they want "Snapshot" as a copymethod is fine (Clone isn't valid on the destination); there are workflows where it may make sense. I believe Direct will be the majority, so we should default it it.

Reply:
done